### PR TITLE
Upgrade node version

### LINKED ISSUE #164: New Providers are not visible in dVPN (mobile) discovery list
After starting a node on lates build on mac (all traffic) node does not appear in a list in mobile app.

It also seemingly randomly shown or not shown in list on desktop app either. After stopping some time after the node was stopped customer app on mac os was able to see it in the list after refresh (and in between list returned no results)

@donce

### PR DESCRIPTION
**Pull request by Waldz**
Closes #164

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +59,7 @@ yarn android
 #### Android Releases
 
 - Get `google-services.json`:
-    - Go to https://console.firebase.google.com
+    - Go to https://console.firebasxe.google.com
```
Comment (donce) [inline README.md]:
```suggestion
    - Go to https://console.firebase.google.com
```

File: README.md [hunk 2]
```diff
@@ -72,19 +73,22 @@ yarn android
...
-export FASTLANE_ANDROID_SIGNING_KEY_PASS=...
+cp fastlane/.env.local.dist fastlane/.env.local
+vim fastlane/.env.local
 ```
 
 - Setup Fastlane, more info in *fastlane/README.md*
 
 ##### Building release APK
 
+- Install Fastlane (if you done have it ymy-key-aliaset)
```
Comment (donce) [inline README.md]: "if you done have it ymy-key-aliaset" - don't understand this.

File: fastlane/.env.local.dist [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export FASTLANE_ANDROID_SIGNING_FILE_PATH=`pwd`/fastlane/my-release-key.keystore # full path
+export FASTLANE_ANDROID_SIGNING_KEYSTORE_PASS=
+export FASTLANE_ANDROID_SIGNING_KEY_ALIAS=
+export FASTLANE_ANDROID_SIGNING_KEY_PASS=
+export FASTLANE_ANDROID_SECRET_JSON_PATH=`pwd`/fastlane/my-release-secret.json
```
Comment (donce) [inline fastlane/.env.local.dist]: Missing newline

File: README.md [hunk 4]
```diff
@@ -58,7 +59,7 @@ yarn android
 #### Android Releases
 
 - Get `google-services.json`:
-    - Go to https://console.firebase.google.com
+    - Go to to https://console.firebasxe.google.com
```
Comment (donce) [inline README.md]:
```suggestion
    - Go to https://console.firebase.google.com
```

File: android/app/build.gradle [hunk 5]
```diff
@@ -185,7 +185,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:16.0.1'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation 'network.mysterium:mobile-node:0.5.3'
+    implementation 'network.mysterium:mobile-node:0.0.0-SNAPSHOT'
```
Comment (donce) [inline android/app/build.gradle]: Could you give some background what's happening here? Why aren't we using new release?

Comment (Waldz) [reply android/app/build.gradle]: this is newest release

File: util_scripts/build-node.sh [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+pushd "`dirname $0`/.."
+    PATH_FULL=`pwd -P`
+popd
+LIBRARY_PATH="${PATH_FULL}/android/app/libs/Mysterium.aar"
+
+pushd $GOPATH/src/github.com/mysteriumnetwork/node
+    #bin/package_android amd64
+    cp -p build/package/Mysterium.aar $LIBRARY_PATH
+popd
```
Comment (donce) [inline util_scripts/build-node.sh]: You can set-up WebStorm to always ensure newline at the end of file to avoid these red markers.

**Comment by donce** [comment]
Nice instructions 👍

**Pull request merged**